test/e2e/switchable-runtime/pages/api/edge.js

```diff
@@ -0,0 +1,7 @@
+export default (req) => {
+  return new Response(`Returned by Edge API Route ${req.url}`)
+}
+
+export const config = {
+  runtime: `experimental-edge`,
```

Comment:
Maybe we should also add a test case when there **is** a dynamic value included, eg. `experimental-${runtime}` and see it falling back to Node.js?